src/Functions/castOrDefault.cpp

```diff
@@ -376,66 +376,475 @@ SELECT accurateCastOrDefault('abc', 'UInt32')
-If the default value is not provided in the second argument, it is assumed to be zero.
-)",
-            .examples{
-                {"Successful conversion", "SELECT toUInt128OrDefault('1', 2::UInt128)", "1"},
-                {"Default value", "SELECT toUInt128OrDefault('upyachka', 123456789012345678901234567890::UInt128)", "123456789012345678901234567890"},
-                {"Implicit default value", "SELECT toUInt128OrDefault('upyachka')", "0"}},
-            .category = FunctionDocumentation::Category::TypeConversion
-        });
+        toUInt128OrDefault_documentation);
+    FunctionDocumentation::Description toUInt256OrDefault_description = R"(
+Like [`toUInt256`](#touint256), this function converts an input value to a value of type [UInt256](../data-types/int-uint.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
+    )";
+    FunctionDocumentation::Syntax toUInt256OrDefault_syntax = "toUInt256OrDefault(expr[, default])";
+    FunctionDocumentation::Arguments toUInt256OrDefault_arguments = {
+        {"expr", "Expression returning a number or a string representation of a number.", {"String", "Int", "Float"}},
+        {"default", "Optional. The default value to return if parsing is unsuccessful.", {"UInt256"}}
+    };
+    FunctionDocumentation::ReturnedValue toUInt256OrDefault_returned_value = {"Value of type UInt256 if successful, otherwise returns the default value if passed or 0 if not.", {"UInt256"}};
+    FunctionDocumentation::Examples toUInt256OrDefault_examples = {
+        {"Successful conversion", "SELECT toUInt256OrDefault('-256', CAST('0', 'UInt256'))", "0"},
+        {"Failed conversion", "SELECT toUInt256OrDefault('abc', CAST('0', 'UInt256'))", "0"}
+    };
+    FunctionDocumentation::Category toUInt256OrDefault_category = FunctionDocumentation::Category::TypeConversion;
+    FunctionDocumentation::IntroducedIn toUInt256OrDefault_introduced_in = {21, 11};
+    FunctionDocumentation toUInt256OrDefault_documentation = {toUInt256OrDefault_description, toUInt256OrDefault_syntax, toUInt256OrDefault_arguments, toUInt256OrDefault_returned_value, toUInt256OrDefault_examples, toUInt256OrDefault_introduced_in, toUInt256OrDefault_category};
+
     factory.registerFunction("toUInt256OrDefault", [](ContextPtr context)
-        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toUInt256OrDefault", std::make_shared<DataTypeUInt256>()); });
+        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toUInt256OrDefault", std::make_shared<DataTypeUInt256>()); },
+        toUInt256OrDefault_documentation);
+
+    FunctionDocumentation::Description toInt8OrDefault_description = R"(
+Like [`toInt8`](#toint8), this function converts an input value to a value of type [Int8](../data-types/int-uint.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
```

Comment:
```suggestion
Like [`toInt8`](#toInt8), this function converts an input value to a value of type [Int8](../data-types/int-uint.md) but returns the default value in case of an error.
If no `default` value is passed then `0` is returned in case of an error.
```